AutonomousDemo/Loop.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace AutonomousDemo
+{
+    public class Loop
+    {
+
+        public void GameLoop(bool answer)
+        {
+            int lastTime = DateTime.Now.Millisecond;
+            
+            while (true)
+            {
+                int start = DateTime.Now.Millisecond;
+                ProcessInput();
+                Update();
+                Render();
+                Thread.Sleep(900);
```

Comment:
Thread.Sleep(...) goes along with the whole 'real time' thing that I recommend you avoid. Your simulation should be able to process a very large amount of simulated time in a very short amount of human time.